### PR TITLE
430 - Remove undesirable heading element

### PR DESCRIPTION
**Pull request by nicksayre**
## Summary

Removes an undesired heading element. Affected text should instead match as regular body copy.

I work for the IIR Team. We are the owners of Resources For Schools.

## Related issue(s)

Related to [vagov-content PR #925](https://github.com/department-of-veterans-affairs/vagov-content/pull/925).

## Testing done

This static site does not have testing.

## Screenshots

_Note: This field is mandatory for UI changes (non-component work should NOT have screenshots)._

Before (Links are wrapped in H4s):
Screenshot 2024-03-06 at 3 44 20 PM
After (Links match regular body copy style):
Screenshot 2024-03-06 at 3 43 44 PM

## What areas of the site does it impact?

Only effects Resources For Schools ([https://www.va.gov/school-administrators/](https://www.va.gov/school-administrators/)).

### Quality Assurance & Testing

- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [x] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [ ] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [ ] Did you login to a local build and verify all authenticated routes work as expected with a test user

### CODE REVIEW

**Review by khenson-oddball — APPROVED** [review]

File: src/site/components/merger-social-sco.html [hunk 1]
```diff
@@ -40,7 +40,6 @@ <h4>{{ subs.subhead }}</h4>
                 <li class="vads-u-margin-bottom--0 vads-u-margin-top--0">
                 {% endif %}
                   {% if navPath == 'Ask questions' %}
```
Comment (maxx1128) [inline src/site/components/merger-social-sco.html:42]: **Todo:** This conditional was only there for the heading element being removed, so it should be removed too.

File: src/site/components/merger-social-sco.html [hunk 2]
```diff
@@ -66,7 +65,6 @@ <h4 class='va-nav-linkslist-list'>
                       {% endif %}
                     </a>
                   {% if navPath == 'Ask questions' %}
```
Comment (maxx1128) [inline src/site/components/merger-social-sco.html:67]: **Todo:** This conditional should be removed here too.

**Review by maxx1128 — CHANGES_REQUESTED** [review]
Two small additional changes and it looks good to me.

**Comment by Andrew565** [comment]
PR has requested changes, opting not to review at this time. - Tier 2/FE CoP

File: src/site/components/merger-social-sco.html (hunk 1 above)
Comment (nicksayre) [reply src/site/components/merger-social-sco.html]: Fixed!

File: src/site/components/merger-social-sco.html (hunk 2 above)
Comment (nicksayre) [reply src/site/components/merger-social-sco.html]: Fixed this one too.

**Review by Andrew565 — APPROVED** [review]

**Review by maxx1128 — APPROVED** [review]

**Pull request merged**